debug.html

```diff
@@ -0,0 +1,285 @@
+      rel="stylesheet"
+    />
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;700&display=swap"
+      rel="stylesheet"
+    />
+
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="stylesheet" href="./css/animate.css" />
+    <link rel="stylesheet" href="./css/navbar.css" />
+    <link
+      href="https://api.mapbox.com/mapbox-gl-js/v2.10.0/mapbox-gl.css"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./css/debug.css" />
+    <!-- Primary Meta Tags -->
+    <meta name="title" content="Real Dev Squad" />
+    <meta
+      name="description"
+      content="Community of developers who collaborate to build Real software."
+    />
+
+    <!-- Open Graph / Facebook Meta Tags -->
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="http://localhost:3001/" />
+    <meta property="og:title" content="Real Dev Squad" />
+    <meta
+      property="og:description"
+      content="Community of developers who collaborate to build Real software."
+    />
+    <meta
+      property="og:image"
+      content="https://raw.githubusercontent.com/Real-Dev-Squad/website-www/develop/img/Real-Dev-Squad-preview.png"
+    />
+
+    <!-- Twitter Meta Tag-->
+    <meta property="twitter:card" content="summary_large_image" />
+    <meta property="twitter:site" content="@RealDevSquad" />
+    <meta property="twitter:title" content="Real Dev Squad" />
+    <meta
+      property="twitter:description"
+      content="Community of developers who collaborate to build Real software."
+    />
+    <meta
+      property="twitter:image"
+      content="https://raw.githubusercontent.com/Real-Dev-Squad/website-www/develop/img/Real-Dev-Squad-preview.png"
+    />
+
+    <script src="/js/analytics.js"></script>
+
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script
+      async
+      src="https://www.googletagmanager.com/gtag/js?id=G-YRG12HRZKB"
+    ></script>
+
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag() {
+        dataLayer.push(arguments);
+      }
+      gtag('js', new Date());
+
+      gtag('config', 'G-YRG12HRZKB');
+    </script>
+  </head>
+  <body>
+    <div class="dropdown hide">
+      <ul class="dropdown-list">
+        <li class="dropdown-item dropdown-link" id="myprofile-option">
+          My Profile
+        </li>
+        <hr class="line" />
+        <li class="dropdown-item dropdown-link" id="signout-option">
+          Sign out
+        </li>
+      </ul>
+    </div>
+    <nav>
+      <div class="hamburger">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
+      <div class="nav-login">
+        <a
+          class="btn-login"
+          href="https://github.com/login/oauth/authorize?client_id=23c78f66ab7964e5ef97"
+        >
+          <button class="btn-login-text">
+            Sign In
+            <img
```

Comment:
Please avoid in-line styling

Reply:
I have changed this in /debug page but usage of inline-styles is throughout the whole page, should I fix that in this pull request it self i should create a new pull request?

Reply:
If you have used the inline styling, then fix all of them in this PR only (just write these styles in classes and add them to the elements) 👍🏻